command/ca/provisionerbeta/remove.go

```diff
@@ -0,0 +1,54 @@
+package provisionerbeta
+
+import (
+	"github.com/smallstep/certificates/ca"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/flags"
+	"github.com/smallstep/cli/utils/cautils"
+	"github.com/urfave/cli"
+)
+
+func removeCommand() cli.Command {
+	return cli.Command{
+		Name:      "remove",
+		Action:    cli.ActionFunc(removeAction),
+		Usage:     "remove a provisioner from the CA configuration",
+		UsageText: `**step beta ca provisioner remove** <name> [**--ca-url**=<uri>] [**--root**=<file>]`,
+		Flags: []cli.Flag{
+			flags.X5cCert,
+			flags.X5cKey,
+			flags.PasswordFile,
+			flags.CaURL,
+			flags.Root,
+		},
+		Description: `**step beta ca provisioner remove** removes a provisioner from the CA configuration.
+
+## EXAMPLES
+
+Remove provisioner by name:
+'''
+$ step beta ca provisioner remove admin-jwk
```

Comment:
To be consistent you might want to use max@smallstep.com in the example, as you've been used that in other examples.

Reply:
That's a subject in my examples, I believe. Not a provisioner name.